why-figjam/index.qmd

```diff
@@ -0,0 +1,152 @@
+---
+title: "Why FigJam for retrospectives"
+description: "Our reasons for using FigJam for our retrospectives."
+date: "2024-10-25"
+categories:
+  - collaboration
+  - retrospectives
+---
+
+::: content-hidden
+Use other decision posts as inspiration to writing these. Leave the
+content-hidden sections in the text for future reference.
+:::
+
+## Context and problem statement
+
+::: content-hidden
+State the context and some background on the issue, then write a
+statement in the form of a question for the problem.
+:::
+
+As part of the design cycle for the Seedcase Project, we do all our
+developments in iteration which generally run for about three weeks. At
+the end of each iteration, we have a retrospective meeting where we
+discuss what went well, what didn't go well, and what we can improve.
+After having worked with an external consultancy, we have been using
+FigJam for our retrospectives. This has been a great tool for us to
+collaborate and share our thoughts and ideas. However, we need to make a
+decision on whether we should continue using FigJam for our
+retrospectives or if we should look for an alternative tool.
+
+## Decision drivers
+
+::: content-hidden
+List some reasons for why we need to make this decision and what things
+have arisen that impact work.
+:::
+
+The decision to look at our retrospective tools was primarily driven by
+the fact that the FigJam board we use is hosted by the consultancy
+company we worked with at the end of 2023. We have been assured that we
+are more than welcome to keep using it, but it is not really a
+sustainable solution. We should have a board where we are the owners,
+and which can't suddenly disappear because someone decides that former
+clients should no longer have access to their accounts. To start with,
+we were convinced that we couldn't use the free version of FigJam, so we
+started looking for alternatives. We collected information on a variety
+of online open access tools, but more importantly, collated a list of
+requirements and features that we need for our retrospectives. Towards
+the end of this exercise, we found out that we could use the free
+version of FigJam, and that it would be easier to continue using it than
+to switch to a new tool.
+
+Having done all the work, we decide to write up our results in this
+post, so that we can refer back to it in the future, should we at some
+point need to revisit our decision. This means that the rest of this
+post won't conform to the usual decision post format, but will instead
+be a summary of our findings.
```

Comment:
```suggestion

```

I'd prefer you did include and compare the other tools, so we don't need to do it again if we do decide to change later, plus, it would give a fair comparison of the options available.